public/app/core/components/ForgottenPassword/ChangePassword.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { FC, SyntheticEvent } from 'react';
 import { Tooltip, Form, Field, Input, VerticalGroup, Button, LinkButton } from '@grafana/ui';
 import { selectors } from '@grafana/e2e-selectors';
-import { submitButton } from './LoginForm';
+import { submitButton } from '../Login/LoginForm';
 interface Props {
   onSubmit: (pw: string) => void;
-  onSkip: (event?: SyntheticEvent) => void;
+  onSkip?: (event?: SyntheticEvent) => void;
```

Comment:
Why is this optional?

Reply:
You have the regular change password and the "admin-skip" change password. Could use children there instead and pass in a Skip button too. What do you think?